### ISSUE TITLE
Home Page Content

### ISSUE THREAD

**Issue by Jamesllllllllll** [CodeCafeCommunity/codecafecommunity.github.io#27]
Having the content of the home page will help us with design. 

Here are some ideas for the content:

- Hero section: name, tagline and CTA to join the discord.
- About us section
- Features section: Network & collaborate with other learners, help new learners, events, etc.
- Highlights: Stats, testimonials, projects.
- Resources: Blogs, guides (how to ask for help in Discord)
- Footer section: another link to join Discord, privacy policy and terms/conditions

**Comment by Cykotech** [comment]
I feel that Issues #38 #39 and #43 encapsulate more specific responsibilities of this issue and because of that, this one could be closed.

**Comment by aedwardg** [comment]
Agreed, between those issues and what was already completed in #35 I think we close this for now and open up more narrowly scoped tickets as needed. We can always refer back to this one in the future if necessary.
